### PR TITLE
DRAFT: PWA: intro to basic tutorial

### PR DESCRIPTION
**Pull request by estelle**
Intro and outline for the intro level tutorial.

Not ready for review: but feedback on whether content is what we want to cover is welcome.

### CODE REVIEW

File: files/en-us/web/progressive_web_apps/tutorials/intro/index.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+### A single code base installable everywhere
+
+Historically, operating system specific applications needed to be written in operating system targeted programming languages, then compiled and packaged based on OS language and application requirement, which differ by operating system, and distributed to users thru downloads, often in proprietary app stores, and then installed by the user.
+
+For an application to be installable on an operating system, such as Windows, iOS, MacOS, Linux, and Android, the application had to be developed in a programming languages, like C#, .Net, Objective C, Swift, Kotlin, Java, or Python. PWAs solve this problem: they use a single code base, written using regular open web technologies.
+
+Historically, in most cases, the application code had to be compiled and packaged using the operating system's SDK into downloadable .exe, .dmg, or other installable files or were made available through a vendor like the Apple App Store, [Goole Play](https://play.google.com/store/apps), [Microsoft Store App](https://apps.microsoft.com/store/apps), or similar. PWAs don't have these requirements.
```
Comment (dawei-wang) [inline files/en-us/web/progressive_web_apps/tutorials/intro/index.md:18]:
```suggestion
Historically, in most cases, the application code had to be compiled and packaged using the operating system's SDK into downloadable .exe, .dmg, or other installable files or were made available through a vendor like the Apple App Store, [Google Play](https://play.google.com/store/apps), [Microsoft Store App](https://apps.microsoft.com/store/apps), or similar. PWAs don't have these requirements.
```

File: files/en-us/web/progressive_web_apps/tutorials/intro/index.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+Like regular web sites, PWA are hosted on and downloable from web servers. They don't require any packaging. They can be seamlessly installed on any operating system without the need of app stores (nor app store approval and fees). PWA don't require any additional programming language knowledge, any proprietary SDK, or even an app store.
```
Comment (dawei-wang) [inline files/en-us/web/progressive_web_apps/tutorials/intro/index.md:20]:
```suggestion
Like regular web sites, PWAs are hosted on and downloadable from web servers. They don't require any packaging. They can be seamlessly installed on any operating system without the need of app stores (nor app store approval and fees). PWAs don't require any additional programming language knowledge, any proprietary SDK, or even an app store.
```

File: files/en-us/web/progressive_web_apps/tutorials/intro/index.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+### Similarities to installed applications
+
+Once installed, PWAs can be made to appear and act similar to other applications installed on the user's device:
+
+- Application window
+
+  - : Similar to other installed applications, depending on PWA settings defined by the developer and set with user preferences, the PWA can be made to open in its own application window.
+
+- Application icon
+  - : PWA display an application icon in the same location as other installed applications installed on the users' operating system: this can be an icon on the homescreen, in the tool bar, in the applications folder, or wherever the devics displays the icons of available applications.
```
Comment (dawei-wang) [inline files/en-us/web/progressive_web_apps/tutorials/intro/index.md:41]:
```suggestion
  - : PWAs display an application icon in the same location as other installed applications installed on the users' operating system: this can be an icon on the homescreen, in the tool bar, in the applications folder, or wherever the device displays the icons of available applications.
```

File: files/en-us/web/progressive_web_apps/tutorials/intro/index.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
+---
+title: Creating your first PWA
+slug: Web/Progressive_web_apps/tutorials/intro
+---
+
+Progressive web applications, known as PWAs, are a way to build applications for all devices using the web technologies we all know and love: HTML, CSS, and JavaScript.
+
+## The benefit of PWA
```
Comment (dawei-wang) [inline files/en-us/web/progressive_web_apps/tutorials/intro/index.md]:
```suggestion
## The benefit of PWAs
```

**Review by dawei-wang — COMMENTED** [review]
A few typos

File: files/en-us/web/progressive_web_apps/tutorials/intro/html/index.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
+---
+title: Basic PWA page template
+slug: Web/Progressive_web_apps/tutorials/intro/html
```
Comment (teoli2003) [inline files/en-us/web/progressive_web_apps/tutorials/intro/html/index.md:3]: Drive-by comment:

Fix capitalization in slugs:
```suggestion
slug: Web/Progressive_web_apps/Tutorials/Intro/HTML```

File: files/en-us/web/progressive_web_apps/tutorials/intro/index.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
+---
+title: Creating your first PWA
+slug: Web/Progressive_web_apps/tutorials/intro
```
Comment (teoli2003) [inline files/en-us/web/progressive_web_apps/tutorials/intro/index.md:3]: Drive-by comment
```suggestion
slug: Web/Progressive_web_apps/Tutorials/Intro
```

**Review by teoli2003 — COMMENTED** [review]
I would even consider renaming Intro to Introduction (or finding a better term)

**Review by captainbrosset — COMMENTED** [review]
Great start. I really like the cats idea, and agree that we should keep this intro tutorial super simple. Excited to see where this PR is going.
